### DISCUSSION TITLE
Logo for dynamic cli

### DISCUSSION THREAD

**Discussion by lainq** [IndianOpenSourceFoundation/dynamic-cli#103] [Ideas]
We need a logo for the dynamic cli 😕

**Comment by GouravSardana** [comment]
@Nuel0 @isaaczara can help here. They are the wonderful designers and @isaaczara even do branding as well.

**Comment by isaaczara** [comment]
Hi Gourav, what is it all ‘bout?

**Reply by GouravSardana** [reply]
@isaaczara It's a command-line package that will use to test the APIs (Application programming interface) and if you got stuck at some error then you can search your error in StackOverflow using this CLI. This will return a list of answers depends upon the upvote and display it in the terminal and you can navigate to find out your answer.
If you want to bookmark any answer then it can also be possible, we have a feature called playbook (just like a bookmark) where you can see your saved answers. Later you can upload your playbook data to some kind of cloud storage (Such as notion).

PS: This notion uploading part is pending right now.

**Reply by GouravSardana** [reply]
@isaaczara @Nuel0  Can you help us?
